api/soap/mantisconnect.wsdl

```diff
@@ -1000,6 +1014,11 @@
     <input message="tns:mc_issue_addRequest"/>
     <output message="tns:mc_issue_addResponse"/>
   </operation>
+  <operation name="mc_issue_report">
+    <documentation>Same as mc_issue_add + mc_issue_attachment_add, except returns full issue data instead of just id.</documentation>
```

Comment:
That is inaccurate, this API will do the 4 steps you indicated in our discussion.

- Search for the issue.
- If found, increment the vote / hit count.
- If not found, add it.
- Add the note + attachment.